pkg/kubernetes/client/v1alpha1/client.go

```diff
@@ -158,16 +216,21 @@ func (c *Client) getConfigFromENV() (config *rest.Config, err error) {
 	kubeConfig := c.getKubeConfigPath(KubeConfigEnvironmentKey)
 	if strings.TrimSpace(k8sMaster) == "" &&
 		strings.TrimSpace(kubeConfig) == "" {
-		return nil, errors.Errorf("failed to get kubernetes config: missing environment variables: atleast one should be set: %s or %s", K8sMasterIPEnvironmentKey, KubeConfigEnvironmentKey)
+		return nil, errors.Errorf(
+			"failed to get kubernetes config: missing ENV: atleast one should be set: {%s} or {%s}",
+			K8sMasterIPEnvironmentKey,
+			KubeConfigEnvironmentKey,
+		)
 	}
 	return c.buildConfigFromFlags(k8sMaster, kubeConfig)
 }
 
-// Clientset returns a new instance of kubernetes clientset
-func (c *Client) Clientset() (*kubernetes.Clientset, error) {
+// Dynamic returns a kubernetes dynamic client capable of invoking operations
+// against kubernetes resources
+func (c *Client) Dynamic() (dynamic.Interface, error) {
 	config, err := c.Config()
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to get kubernetes clientset")
+		return nil, errors.Wrap(err, "failed to provide dynamic client")
 	}
-	return c.getKubernetesClientset(config)
+	return c.getKubernetesDynamicClientSet(config)
```

Comment:
let us call this as DynamicClient.
Can we rename this to `getKubernetesDynamicClient`

Reply:
Done